src/style.css

```
@@ -0,0 +1,79 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, sans-serif;
+
+  --color-white: #fff;
+  --color-silver: #e1e7ed;
+  --color-gray: #3d4e61;
+  --color-dark: rgba(61, 78, 97, 0.1);
+}
+
+.form1,
+.form2 {
+  position: relative;
+  margin-top: 20px;
+}
+
+.form__input {
+  box-sizing: border-box;
+  padding-left: 62px;
+  width: 100%;
+  height: 70px;
+  background: var(--color-white);
+  border: 1px solid var(--color-silver);
+  box-shadow: 0 1px 8px var(--color-dark);
+  border-radius: 4px;
+
+  font-family: inherit;
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 22px;
+  color: var(--color-gray);
+}
+
+.input--small {
+  padding-left: 33px;
+  height: 42px;
+
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.form__input:hover {
+  background: var(--color-white);
+  box-shadow: 0 3px 8px var(--color-dark);
+}
+
+.form__input:focus {
+  background: linear-gradient(180deg, var(--color-white) 0%, #f6f6f7 100%);
+  outline: none;
+}
+
+.form1 input::placeholder {
+  font-weight: normal;
+  color: var(--color-gray);
+}
+
+.form2 input::placeholder {
+  font-weight: normal;
+  color: var(--color-gray);
+}
```

you have already have a selector for input 
Also these selectors are the same
```suggestion
.form_input::placeholder {
  font-weight: normal;
  color: var(--color-gray);
}
```